neo/io/nwbio.py

```diff
@@ -434,10 +435,15 @@ def write_all_blocks(self, blocks, **kwargs):
         if "session_description" not in annotations:
             annotations["session_description"] = blocks[0].description or self.filename
             # todo: concatenate descriptions of multiple blocks if different
+        
         if "session_start_time" not in annotations:
-            raise Exception("Writing to NWB requires an annotation 'session_start_time'")
+            annotations["session_start_time"] = blocks[0].rec_datetime
+            if annotations["session_start_time"]==None:
+                annotations["session_start_time"] = datetime.now()
```

Comment:
I don't think we should use `datetime.now()`, as it is probably not the correct  session start time. We should raise an Exception instead